bin/server.js

```diff
@@ -41,11 +42,23 @@ var launch = function (profile) {
   var maxInstanceLife = cfg.get('provisioner:maxInstanceLife');
 
   // Create InfluxDB connection for submitting statistics
-  var influx = new base.stats.Influx({
-    connectionString: cfg.get('influx:connectionString'),
-    maxDelay: cfg.get('influx:maxDelay'),
-    maxPendingPoints: cfg.get('influx:maxPendingPoints'),
-  });
+  var influx;
+  if (useNullInfluxDrain) {
+    var influx = new base.stats.NullDrain();
+
+    // Ugly*1000 but otherwise, it'll be really hard to get a reference to this
+    // drain aiui, which we need to get the number of pending points to make
+    // sure that the stat was submitted for real
+    module.exports.influx = influx;
```

Comment:
Don't use globals unless strictly necessary, you can set it on the `server` object created by `app.createServer();`.